database/seed.js

```diff
@@ -0,0 +1,44 @@
+const db = require("./index.js");
+const Listing = require("./Listing.js");
+
+const listings = [
+  {
+    documentId: 1,
+    listingId: 1,
+    assets: [
```

Comment:
Clarissa, check out my seed.js doc to see how I selected n random photos from an array of 15 total photos so that there's a variety in the number of photos a property might have.

Reply:
It's on my to-do list! Just setting up the basics now.